Doc/whatsnew/3.15.rst

```diff
@@ -155,3 +155,30 @@ Deprecated C APIs
 Removed C APIs
 --------------
 
+* Python initialization functions:
+
+  * :c:func:`!Py_GetExecPrefix`:
+    use :c:func:`PyConfig_Get("base_exec_prefix") <PyConfig_Get>`
+    (:data:`sys.base_exec_prefix`) instead.
+    Use :c:func:`PyConfig_Get("exec_prefix") <PyConfig_Get>`
+    (:data:`sys.exec_prefix`) if :ref:`virtual environments <venv-def>`
+    need to be handled.
+  * :c:func:`!Py_GetPath`:
+    use :c:func:`PyConfig_Get("module_search_paths") <PyConfig_Get>`
+    (:data:`sys.path`) instead.
+  * :c:func:`!Py_GetPrefix`:
+     use :c:func:`PyConfig_Get("base_prefix") <PyConfig_Get>`
+     (:data:`sys.base_prefix`) instead.
+     Use :c:func:`PyConfig_Get("prefix") <PyConfig_Get>`
+     (:data:`sys.prefix`) if :ref:`virtual environments <venv-def>`
+     need to be handled.
+  * :c:func:`!Py_GetProgramFullPath`:
+     use :c:func:`PyConfig_Get("executable") <PyConfig_Get>`
+     (:data:`sys.executable`) instead.
+  * :c:func:`!Py_GetProgramName`:
+    use :c:func:`PyConfig_Get("executable") <PyConfig_Get>`
+    (:data:`sys.executable`) instead.
+  * :c:func:`!Py_GetPythonHome`:
+    use :c:func:`PyConfig_Get("home") <PyConfig_Get>` or the
+    :envvar:`PYTHONHOME` environment variable instead.
+
```

Comment:
PyConfig_Get() is new, it was added in Python 3.14. You may refer to the pythoncapi-compat project which provides this function on Python 3.13 and older.